Sources/_SwiftSyntaxMacros/CodeItemMacro.swift

```diff
@@ -16,5 +16,5 @@ public protocol CodeItemMacro: FreestandingMacro {
   static func expansion(
     of node: MacroExpansionDeclSyntax,
     in context: inout MacroExpansionContext
-  ) throws -> [CodeBlockItemSyntax]
+  ) throws -> CodeBlockItemListSyntax
```

Comment:
I discussed this with @DougGregor yesterday. If you are intending to merge these items with other items, this should return an array IMO because arrays are easier to work with (you can append, insert, delete, …). IMO you should only create a `CodeBlockItemListSyntax` once you think the item list is complete.

(similar in DeclarationMacros.swift)

Reply:
Good point! In that case this PR is no longer necessary. `DeclarationMacro` still returns decls, `ExpressionMacro` still returns exprs, and `CodeItemMacro` returns `[CodeBlockItemSyntax]`